games-simulation/openrct2/openrct2-0.0.7.ebuild

```diff
@@ -81,28 +81,23 @@ src_configure() {
 		-DWITH_TESTS="$(usex test)"
 	)
 
+	# workaround a missing definition for speexdsp
+	append-cppflags -DHAVE_STDINT_H
```

Comment:
I am not really sure whether this should go in here. It is not going to hurt, but at least we should try to get it fixed properly.
I haven't checked it in too much detail, but I think it is a bug in speex. I will see into in this weekend.